content/en/docs/releases/bugs/index.md

```diff
@@ -56,7 +56,7 @@ reference](/docs/reference/commands/istioctl/#istioctl-bug-report).
 If you are unable to use the `bug-report` command, please attach your own archive
 containing:
 
-* Output of istioctl analyze:
+* Output of `istioctl` analyze:
```

Comment:
```suggestion
* Output of `istioctl` analyze:
```
It's not suggested to include two languages in a PR, which need reviewers from zh and en teams and will postpone your process to merge.